app/queries/get_content_collection.rb

```diff
@@ -1,11 +1,12 @@
 module Queries
   class GetContentCollection
-    attr_reader :content_format, :fields
+    attr_reader :content_format, :fields, :locale
 
-    def initialize(content_format:, fields:, publishing_app: nil)
+    def initialize(content_format:, fields:, publishing_app: nil, locale: nil)
```

Comment:
Technically those aren't equivalent, e.g.

```ruby
def foo(bar: "en")
  puts bar.inspect
end

foo(bar: nil)
# nil
```

Named parameter defaults are ignored if you pass in `nil` explicitly, which is the case here because that's coming from `params[:locale]` in the controller.